.storybook/components/DesignTokens/Legacy/LegacyTypography.stories.mdx

```diff
@@ -19,30 +19,42 @@ import styles from './TypographyTable.module.css';
       <td>
         <Heading size="h1">Heading 1</Heading>
       </td>
-      <td>24 / 32</td>
```

Comment:
I'm wondering if we should just remove these legacy token stories as the tokens themselves are changed.

Reply:
Makes sense to me

Reply:
yeah that sounds good to me! or maybe we can just show the 2 deprecated legacy sizes?

Reply:
>or maybe we can rename the story title to "Legacy Typography Sizes" and only show the 2 deprecated small sizes?

Oo good idea, especially since I left these two out of the main ramp doc because they're deprecated and I don't want to encourage their usage.